tests/installation/bootloader.pm

```diff
@@ -29,6 +29,7 @@ use warnings;
 
 use testapi;
 use lockapi 'mutex_wait';
+use autoyast qw(expand_agama_profile);
```

Comment:
Do you use this 'expand_agama_profile' in this file?

Reply:
Cool, you found it, please see https://suse.slack.com/archives/C02CANHLANP/p1737026770556229

Don't know why, but it can work :)